dbms/src/test/java/org/polypheny/db/misc/DataMigratorTest.java

```diff
@@ -55,14 +66,14 @@ public void basicTest() throws SQLException {
                 try {
                     statement.executeUpdate( "INSERT INTO datamigratortest VALUES (1,5,'foo')" );
 
-                    // Add data store
+                    // Add data storeId
                     statement.executeUpdate( "ALTER ADAPTERS ADD \"store1\" USING 'Hsqldb' AS 'Store'"
                             + " WITH '{maxConnections:\"25\",trxControlMode:locks,trxIsolationLevel:read_committed,type:Memory,tableType:Memory,mode:embedded}'" );
                     connection.commit();
                     // Add placement
                     statement.executeUpdate( "ALTER TABLE \"datamigratortest\" ADD PLACEMENT ON STORE \"store1\"" );
 
-                    // Remove placement on initial store
+                    // Remove placement on initial storeId
```

Comment:
Looks like a find and replace error

Reply:
https://github.com/polypheny/Polypheny-DB/pull/464#discussion_r1520476075